src/haddock/modules/topology/topocg/defaults.yaml

```diff
@@ -1 +1,182 @@
-{}
+cgffversion:
+  default: 2
+  type: integer
+  min: 2
+  max: 3
+  choices:
+    - 2
+    - 3
+  title: MARTINI Force-field version
+  short: Force-field version for the coarse-grain mapping and parameters
+  long: CNS, the computational engine used by HADDOCK can generate a lot of output
+    messages. This parameter controls the verbosity of CNS (verbose, normal or quiet).
+  group: module
+  explevel: expert
+autohis:
```

Comment:
Is this relevant for CG mode? Do we have various HIS states in Martini and do we check for it?

Reply:
We have multiple states of HIS in Martini, a protonated (charged) and non protonated (neutral) state. We could add implement it

Reply:
This should then be based on the protonation of HIS from the AA model, looking at which hydrogen exist on the HIS ring. Topo and parameters will have to be added for those histidines which have most likely different names.

Reply:
Yes, I was just thinking about this !